test/allowlist_for_publicAPI.json

```diff
@@ -622,7 +622,7 @@
     "OrderedDict"
   ],
   "torch.nn.qat.dynamic.modules.linear": [
-    "activation_is_memoryless"
+    "_activation_is_memoryless"
```

Comment:
I think we can remove these after we made them private

Reply:
yeah

Reply:
i remove them at the end of the stack since there's a lot that can get removed

Reply:
I remove them in the final PR